### PR TITLE
Fire EVT_WEBVIEW_NAVIGATING when redirecting

### PR DESCRIPTION
**Pull request by josuegomes**
wxWebView with IE backend was not firing EVT_WEBVIEW_NAVIGATING when redirecting.

By enabling DOCHOSTUIFLAG_ENABLE_REDIRECT_NOTIFICATION flag the event BEFORENAVIGATE2 is fired thus triggering EVT_WEBVIEW_NAVIGATING

See https://forums.wxwidgets.org/viewtopic.php?f=1&t=44969&sid=aef4e95fab6b8b7234c0dfb6134bbf9d
and see https://trac.wxwidgets.org/ticket/17593

### CODE REVIEW

File: src/msw/webview_ie.cpp [hunk 1]
```diff
@@ -34,6 +34,10 @@
 #include <initguid.h>
 #include <wininet.h>
 
+#ifndef DOCHOSTUIFLAG_ENABLE_REDIRECT_NOTIFICATION
+#define DOCHOSTUIFLAG_ENABLE_REDIRECT_NOTIFICATION  0x04000000
+#endif
```
Comment (catalinr) [inline src/msw/webview_ie.cpp]: This should be defined in `include/wx/msw/webview_missing.h` along with the other flags.

File: include/wx/msw/webview_missing.h [hunk 2]
```diff
@@ -944,4 +945,3 @@ struct wxIMarkupServices : public IUnknown
 #endif
 
 /* end of WinInet.h */
-
```
Comment (MaartenBent) [inline include/wx/msw/webview_missing.h]: You should not remove the last line.

**Comment by vadz** [comment]
Just checking: was this closed intentionally?

**Comment by josuegomes** [comment]
Yes, it was.

**Comment by vadz** [comment]
So is this change not needed at all or did you find a better way to do it?

**Comment by josuegomes** [comment]
I'm reviewing and will submit a new PR later.

**Pull request closed without merging**